packages/helpers/qrcode-modal/src/browser/components/Modal.tsx

```diff
@@ -25,6 +25,7 @@ function Modal(props: ModalProps) {
   const displayProps = {
     text: props.text,
     uri: props.uri,
+    navigate: props.qrcodeModalOptions ? props.qrcodeModalOptions.navigate : true,
```

Comment:
The default here if qrcodeModalOptions is unspecified is `true`.  Arguably, for maximum compatibility, that should be `false`, but it seems rather harmless to me to always do the iframe trick.